website/docs/cloud-docs/workspaces/settings/run-tasks.mdx

```diff
@@ -14,7 +14,7 @@ You can manage run tasks through the UI as detailed below or through the [Run Ta
 
 ## Requirements
 
-**Terraform Version** - You can only assign run tasks to workspaces that use a Terraform version of 0.12 and later. Downgrading a workspace with existing run tasks to use a prior Terraform version will not result in an error, but Terraform Cloud will no longer trigger the run tasks during plan and apply operations.
+**Terraform Version** - You can only assign run tasks to workspaces that use a Terraform version of 1.1.9 and later. Downgrading a workspace with existing run tasks to use a prior Terraform version will not result in an error, but Terraform Cloud will no longer trigger the run tasks during plan and apply operations.
```

Comment:
```suggestion
**Terraform Version** - You can only assign run tasks to workspaces that use a Terraform version of 1.1.9 and later. Downgrading a workspace with existing run tasks to use a prior Terraform version does not result in an error, but Terraform Cloud no longer triggers the run tasks during plan and apply operations.
```

A couple of minor wording change suggestions to maintain present tense - "will not result" to "does not result" and "will not longer trigger" to "no longer triggers".

Reply:
The future tense here was chosen by another contributor long before this PR. That being said, it seems justified, as it implies a conditional sentence of the form: `if you downgrade, then an error will not be displayed`

Reply:
To improve readability we can break up the sentence like this:
`Downgrading a workspace with existing run tasks to use a prior Terraform version will not result in an error. Instead, Terraform Cloud will no longer trigger run tasks during plan and apply operations.`

Reply:
I understand where you're coming from about the future tense, but our internal style guidelines dictate that we do not use future tense in cases like this. 

In thinking of other ways to rework the 2nd sentence to improve readability, I do actually agree with you about breaking up the sentence. It makes the text clearer and more understandable. 

So what if we tried rewording the sentences to the following:

"You can downgrade a workspace with existing runs to use a prior Terraform version without causing an error. However, Terraform Cloud no longer triggers the run tasks during plan and apply operations."

Reply:
I have updated the sentence with your suggestion. Thank you for the review.